packages/affiliates/test/datasets/dapp-mining-set1/logs_0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5.txt

```diff
@@ -0,0 +1,94 @@
+{"log_index":204,"transaction_hash":"0x2f23b55607432961f4659065969fbb7094ab75b086acda6b6a1b73756ea93a11","transaction_index":131,"address":"0xe4256c47a3b27a969f25de8bef44eca5f2552bd5","data":"0x","topics":["0x4b82aa16e071a61de1a6b9aeec9edab0356331f8122c78683b469ac8e685dabc","0x000000000000000000000000049355e4380f8db88cb8a6ec0426b1a1a3560c67","0x00000000000000000000000000000000000000000000000003782dace9d90000","0x0000000000000000000000000000000000000000000000056bc75e2d63100000"],"block_timestamp":{"value":"2020-12-18T17:01:55.000Z"},"block_number":11478405,"block_hash":"0x891f28bd91de00c369c7425ff592d4351cad56e96aa1604a49ba68f0247ebd59"}
```

Comment:
It looks like json but its not a valid json file since the objects are not wrapped in an array or object. I didnt want to give the impression that you could import it into node without some pre-processing.